src/Csv.php

```diff
@@ -378,11 +378,11 @@ public function parse($input = null, $offset = null, $limit = null, $conditions
 
         if (strlen($input) <= PHP_MAXPATHLEN && is_readable($input)) {
             $this->file = $input;
-            $this->data = $this->parse_file();
+            $this->data = $this->parse_file_internal();
```

Comment:
What do you think about the name `_parse_string()` instead? Proper used for private methods but here often used for protected methods, too.

Reply:
You're right, that is commonly used for non-public stuff. I didn't think about that. When I have time, I'll add the @internal annotation, too.

Reply:
I renamed the functions to conform with the others. I decided against **@internal** because this is what `protected` should already tell. The annotation would not anything new.